public/app/features/alerting/unified/components/contact-points/useContactPoints.tsx

```diff
@@ -410,10 +410,12 @@ export const useCreateContactPoint = ({ alertmanager }: BaseAlertmanagerArgs) =>
         oldConfig: config,
         alertManagerSourceName: alertmanager,
       })
-    ).then(() => {
-      dispatch(alertingApi.util.invalidateTags(['AlertmanagerConfiguration', 'ContactPoint', 'ContactPointsStatus']));
-      dispatch(generatedReceiversApi.util.invalidateTags(['Receiver']));
-    });
+    )
+      .unwrap()
```

Comment:
This line is the actual fix - the rest is all tests/making sure we aren't throwing an error in the component/tests

If you comment this `unwrap` out, the new test should fail